packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -945,6 +945,41 @@
       <protected>false</protected>
       <shortDescription>Show less</shortDescription>
     </labels>
+    <labels>
```

Comment:
Hey @acarpentier01!
Can you help me review these labels before merging this PR?